### PR TITLE
improve(affiliates): dappmining - week1 rewards for yd-eth-mar21 contract

### LINKED ISSUE #2405: Run Dapp Mining Rewards for YD-ETH-MAR21 for week of 1/12
**Why**
Dapp mining rewards need to be run from 2021-01-04 through 2021-01-11 23:00 UTC. This is the first dapp mining payout being issued, running on 2 active emps. There are only 2 participants, the opendao team and the uma team. 

Whitelist:
- 0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35
- 0x9a9dcd6b52B45a78CD13b395723c245dAbFbAb71

Contract Name | EMP Address   
-- | -- 
YD-BTC-MAR21| 0x1c3f1A342c8D9591D9759220d114C685FD1cF6b8
YD-ETH-MAR21 | 0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5

We will be giving out half of the developer mining rewards for these contracts towards dapp mining. Seen in PR #2400

### PR DESCRIPTION
**Pull request by daywiss**
**Motivation**

#2405

**Summary**

First week of dapp mining payouts for the yd-eth-mar21 contract found at `0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5`.
Rewards are based off of dev mining rewards for this contract found in PR #2400. Based on rewards for this contract the total Dapp mining rewards will be ` Math.floor(19617.080282865066141511 * .3) = 5885`. By default untagged rewards will assumed to be coming from UMA address at `0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35`.

**Details**
To reproduce results.
Create a config file called `dappmining.json`
```
{
    "empAddress": "0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5",
    "startTime": 1609718400000,
    "endTime": 1610323200000,
    "totalRewards": 5885,
    "whitelist": ["0x9a9dcd6b52B45a78CD13b395723c245dAbFbAb71"],
    "defaultAddress": "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"
  }
```

And run within the affiliates package with:
`node app/DappMiningRewards dappmining.json`

**Issue(s)**

Fixes #2405

### CODE REVIEW

File: packages/affiliates/payouts/dappmining/2021-01-04-0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "empAddress": "0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5",
+    "startTime": 1609718400000,
+    "endTime": 1610323200000,
+    "totalRewards": 5885,
+    "whitelist": ["0x9a9dcd6b52B45a78CD13b395723c245dAbFbAb71"],
+    "defaultAddress": "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"
+  },
+  "startBlock": {
```
Comment (smb2796) [inline packages/affiliates/payouts/dappmining/2021-01-04-0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5.json:10]: Same comment as on the other pr, I think this timeframe should match up to the developer mining window.

**Review by smb2796 — COMMENTED** [review]
Thanks for running this @daywiss! Big moment with our first dapp mining payout. 

One comment - the rewards calculation window looks off to me when compared to the developer mining timestamps/block numbers.

**Review by smb2796 — APPROVED** [review]
Thanks for fixing the block range. LGTM!

**Comment by hlambur** [comment]
Sent with tx: https://etherscan.io/tx/0xbecb79b699eec9073edaeeb2a4783821d2ef7c0f6249f97db0975c206f6e7ca6

**Pull request merged**